docs/AUTO_UPDATE_PROJECTS.md

```diff
@@ -0,0 +1,121 @@
+# Auto-Update Projects Workflow
+
+This document explains the automated project data generation system implemented for the indietasten.github.io website.
+
+## Overview
+
+The system automatically fetches public repository data from the InDieTasten GitHub organization and generates a structured YAML file containing project information. This eliminates the need to manually maintain the projects list and ensures it stays up-to-date with the latest repository information.
+
+## How It Works
+
+### 1. Scheduled Execution
+- **Daily at 2:00 AM UTC**: The workflow automatically runs to check for updates
+- **Manual trigger**: Can be triggered manually from GitHub Actions tab
+
+### 2. Data Collection
+The workflow fetches the following data for each public repository:
+- Repository name, description, and URL
+- Star count, fork count, and primary language
+- Repository topics (used as tags)
+- Creation and last update timestamps
+- Homepage/website URL
+- Archived status
+- OpenGraph social preview image
+
+### 3. Smart Status Classification
+Projects are automatically classified based on activity:
+- **Done**: Repositories with stars or recent activity
```

Comment:
Repositories with stars shouldn't be done.

Please only mark repositories as done, when they have the tag "status-done".